src/appleseed/renderer/modeling/camera/camera.h

```diff
@@ -173,8 +173,6 @@ class APPLESEED_DLLSYMBOL Camera
     float               m_shutter_open_time_interval;
     float               m_normalized_open_end_time;
     float               m_normalized_close_start_time;
```

Comment:
I also mentioned normalized times, they are implementation specific too. Sorry if that wasn't clear.

Reply:
Hadn't read your message until the end :) Pushing a new commit with a few renamings as well in an attempt to reduce verbosity somewhat.